### ISSUE TITLE
ClearML support for configuration

### ISSUE THREAD

**Issue by pyb0924** [pytorch/ignite#2888]
## ❓ Questions/Help/Support

Hello! I'd like to know how to transfer the ClearML **configuration** logs from pure PyTorch to PyTorch-Ignite, that is:
```
task = Task.init("project-name", task_name="task-name")
task.connect_configuration(config) # or task.connect({k: v for k, v in config.items()})
```
In official docs, I only found `ClearMLLogger` class which implemented `Task.init()` inside, but with no support for `Task.connect()`. Is there any good solution to this problem in current verison of release? 

Thanks!

**Comment by vfdev-5** [comment]
@pyb0924 thanks for asking! At the moment we do not expose task publicly from `ClearMLLogger`. There is a private argument `ClearMLLogger._task` but it is not advised to use it... In our examples, we just import `Task` from clearml package directly: 
https://github.com/pytorch/ignite/blob/ad37a17004ceaf73606f04125f77bdc3705a935c/examples/contrib/cifar10/main.py#L49-L68

I agree that it could be helpful to expose the task from `ClearMLLogger`. Actually, we could do 2 things:
1) Like TensorboardLogger https://github.com/pytorch/ignite/blob/ad37a17004ceaf73606f04125f77bdc3705a935c/ignite/contrib/handlers/tensorboard_logger.py#L163-L164 such that we can pass through directly to [`ClearMLLogger.clearml_logger`](https://github.com/pytorch/ignite/blob/ad37a17004ceaf73606f04125f77bdc3705a935c/ignite/contrib/handlers/clearml_logger.py#L152)
2) Provide new `ClearMLLogger.get_task()` method to return `ClearMLLogger._task`.

I'll mark this topic as "help wanted" and "good first issue" if someone would like to help with it.

**Comment by pyb0924** [comment]
@vfdev-5 Thanks for your clear answer!
But there is still a slight question: I can directly import `Task` from clearml package directly, but suppose I want to simultaneously use `ClearMLLogger` during my training , I have to do like this:
```Python
task = Task.init(project_name=project_name, task_name=task_name)
task.connect_configuration(config)
clearml_logger = ClearMLLogger(project_name=project_name, task_name=task_name) 
clearml_logger.attach_output_handler(
    trainer,
    event_name=Events.ITERATION_COMPLETED(every=log_every),
    tag="training",
    output_transform=lambda loss: {"batchloss": loss},
)
...
```
So, is there any potential conflict produced by this kind of duplicated code?

Thanks!

**Comment by pyb0924** [comment]
By the way, I've tried in one of my own project but raise no conflict. It's just a supposed question to check the answer in principle. Thanks!

**Comment by vfdev-5** [comment]
> So, is there any potential conflict produced by this kind of duplicated code?

yeah, if `task` is already created in the user code, then we fetch current task inside `ClearMLLogger` and use it if it exists. So, basically, there is no conflict.

**Comment by pyb0924** [comment]
Really appreciate to your answering!
